reference-data-proxy/src/v1/controllers/acbs.controller.ts

```diff
@@ -77,6 +77,34 @@ export const findOne = async (req: Request, res: Response) => {
   return res.status(500).send();
 };
 
+const createAcbsRecord = async (deal: any, bank: any) => {
+  if (deal) {
+    const response = await axios({
+      method: 'post',
+      url: `${acbsFunctionUrl}/api/orchestrators/acbs`,
+      data: {
+        deal,
+        bank,
+      },
+    }).catch((err: any) => err);
+    return response;
+  }
+  return {};
+};
+
+export const createAcbsRecordPOST = async (req: Request, res: Response) => {
+  if (req) {
```

Comment:
do we need this check here? I assume that this function is called through an API call. in which case, we should always have a request